lib/__tests__/Autocomplete-test.js

```diff
@@ -11,6 +12,10 @@ import Autocomplete from '../Autocomplete';
 import { getStates, matchStateToTerm, sortStates, styles } from '../utils'
 
 chai.use(chaiEnzyme());
+chai.use(spies);
+
+var onFocusSpy = chai.spy()
+var onBlurSpy = chai.spy()
```

Comment:
These spies don't seem to be reset between each test block, so right now they're only suitable for one-time use. Perhaps a better solution would be to pass them in via `extraProps` to `AutocompleteComponentJSX`, paired with `beforeEach` to reset them for each test test within your block.